src/GitHub.VisualStudio/Views/Dialog/RepositoryCloneView.xaml

```diff
@@ -2,33 +2,28 @@
                                   xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                                   xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                                   xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-                                  xmlns:cache="clr-namespace:GitHub.UI.Helpers;assembly=GitHub.UI"
                                   xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-                                  xmlns:helpers="clr-namespace:GitHub.Helpers;assembly=GitHub.UI"
+                                  xmlns:ghfvs="https://github.com/github/VisualStudio"
                                   xmlns:local="clr-namespace:GitHub.VisualStudio.Views.Dialog"
                                   xmlns:prop="clr-namespace:GitHub.VisualStudio.UI;assembly=GitHub.VisualStudio.UI"
```

Comment:
Is this one still required?

Reply:
Ah or is it one of the `Resources` disambiguations you mentioned are still needed?

Reply:
Yep, that's a `Resource` disambiguation.